week1/3-saturday/homework/exercise9/exercise9.js

```diff
@@ -0,0 +1,19 @@
+function cipher(str, offset) {
+    let alphabet = ["A","B","C","D","E","F",
+    "G","H","I","J","K","L","M","N","O","P","Q","R",
+    "S","T","U","V","W","X","Y","Z"];
+    let ciphered = "";
+    for(let i=0; i<str.length; i++) {
+        let charToCipher = str[i];
+        if(alphabet.includes(charToCipher.toUpperCase())) {
+            let position = alphabet.indexOf(charToCipher.toUpperCase());
+            let newPosition = (position + offset)%26;
+            ciphered = ciphered.concat(alphabet[newPosition]);
+        } else {
+            ciphered = ciphered.concat(str[i]);
```

Comment:
Nice! You could also use an array to collect the characters and then use Array.join.